OpenRA.Mods.Common/Widgets/VideoPlayerWidget.cs

```diff
@@ -46,7 +46,14 @@ public void Load(string filename)
 			if (filename == cachedVideoFileName)
 				return;
 
-			var video = VideoLoader.GetVideo(Game.ModData.DefaultFileSystem.Open(filename), true, Game.ModData.VideoLoaders);
+			var stream = Game.ModData.DefaultFileSystem.Open(filename);
+			if (stream.Length == 0)
```

Comment:
Wouldn't it make sense to have this check as part of `IsWsa` and `IsWestwoodVqa`?

Reply:
No, those functions don't have the filename for logging.

Reply:
`VideoLoader.GetVideo` would return null which means we could still log here.

Reply:
Okay done as that would cover more crashes.